libc/CMakeLists.txt

```diff
@@ -225,6 +225,15 @@ else()
   set(LIBC_INSTALL_INCLUDE_DIR ${CMAKE_INSTALL_INCLUDEDIR})
 endif()
 
+if(LIBC_TARGET_TRIPLE)
```

Comment:
Can you comment a bit about the logic where (i.e. why checking TARGET_TRIPLE)

Reply:
That's the existing logic  that's been moved, so it's not related to the patch. But `LIBC_TARGET_TRIPLE` is set whenever building using https://libc.llvm.org/full_cross_build.html#runtimes-cross-build.

Reply:
The intention is to install these things in subdirectories, like `libcxx` and `libc` end up in `lib/x86_64-unknown-linux-gnu`.